### PR TITLE
add governator-sdk

### PR DESCRIPTION
**Pull request by zberwaldt**
Feel like this is going to take a while to fully replace everything.

### CODE REVIEW

**Comment by wpmonty** [comment]
Restating from Monday's meeting: I'd like to keep this initial PR small as I know it could spiral. Let me know when you reach a stopping point and follow up with a list of how you'd like to complete the remainder of this migration.

**Comment by wpmonty** [comment]
./pages/community/[serverId]/polls/results/[pollId].tsx:59:70
Type error: Type 'string | { data: string; }' is not assignable to type 'string | undefined'.
  Type '{ data: string; }' is not assignable to type 'string'.

File: components/polls/DisplayPollResults.tsx [hunk 1]
```diff
@@ -195,6 +196,13 @@ const ResultBlock: React.FC<DisplayPollResultsProps> = ({pollData, voteData, tot
 
 type BlockHeight = Array<{chain_id: string, block: string}>
 const blockHeightsTable: React.FC<BlockHeight> = (blockHeight) => {
+
+    if (!blockHeight) return (<Spinner size='xl' color='blue.500' />)
```
Comment (wpmonty) [inline components/polls/DisplayPollResults.tsx:200]: this need to be in here? Seems slightly unrelated

Comment (zberwaldt) [reply components/polls/DisplayPollResults.tsx:200]: I was getting a build error, as one of my polls wasn't getting back an array of chains with blockheights. Just a single block number.

Comment (zberwaldt) [reply components/polls/DisplayPollResults.tsx:200]: Maybe this will no longer be an issue, but just wanted to make sure I handled it just in case.

**Review by wpmonty — APPROVED** [review]

**Pull request merged**